resolvers.js

```diff
@@ -0,0 +1,236 @@
+    organism(ds) { return dsField(ds, 'organism'); },
+    polarity(ds) { return dsField(ds, 'polarity').toUpperCase(); },
+    ionisationSource(ds) { return dsField(ds, 'ionisationSource'); },
+    maldiMatrix(ds) { return dsField(ds, 'maldiMatrix'); },
+    
+    submitter(ds) {
+      return ds.metadata.Submitted_By.Submitter;
+    },
+    
+    principalInvestigator(ds) {
+      return ds.metadata.Submitted_By.Principal_Investigator;
+    },
+    
+    analyzer(ds) {
+      const msInfo = ds.metadata.MS_Analysis;
+      return {
+        'type': msInfo.Analyzer,
+        'rp': msInfo.Detector_Resolving_Power
+      };
+    },
+    
+    /* annotations(ds, args) {
+     args.datasetId = ds.id;
+     return esSearchResults(args);
+     } */
+  },
+  
+  Annotation: {
+    id(hit) {
+      return hit._id;
+    },
+    
+    sumFormula(hit) {
+      return hit._source.sf;
+    },
+  
+    possibleCompounds(hit) {
+      const ids = hit._source.comp_ids.split('|');
+      const names = hit._source.comp_names.split('|');
+      let compounds = [];
+      for (var i = 0; i < names.length; i++) {
+        let id = ids[i];
+        let infoURL;
+        if (hit._source.db_name == 'HMDB') {
+          id = sprintf.sprintf("HMDB%05d", id);
+          infoURL = `http://www.hmdb.ca/metabolites/${id}`;
+        } else if (hit._source.db_name == 'ChEBI') {
+          id = "CHEBI:" + id;
+          infoURL = `http://www.ebi.ac.uk/chebi/searchId.do?chebiId=${id}`;
+        } else if (hit._source.db_name == 'SwissLipids') {
+          id = sprintf.sprintf("SLM:%09d", id);
+          infoURL = `http://swisslipids.org/#/entity/${id}`;
+        } else if (hit._source.db_name == 'LIPID_MAPS') {
+          infoURL = `http://www.lipidmaps.org/data/LMSDRecord.php?LMID=${id}`;
+        }
+      
+        compounds.push({
+          name: names[i],
+          imageURL: `http://${config.MOL_IMAGE_SERVER_IP}/mol-images/${hit._source.db_name}/${id}.svg`,
+          information: [{database: hit._source.db_name, url: infoURL}]
+        });
+      }
+      return compounds;
+    },
+  
+    adduct: (hit) => hit._source.adduct,
+  
+    mz: (hit) => parseFloat(hit._source.mz),
+  
+    fdrLevel: (hit) => hit._source.fdr,
+  
+    msmScore: (hit) => hit._source.msm,
+  
+    rhoSpatial: (hit) => hit._source.image_corr,
+  
+    rhoSpectral: (hit) => hit._source.pattern_match,
+  
+    rhoChaos: (hit) => hit._source.chaos,
+  
+    dataset(hit) {
+      return {
+        id: hit._source.ds_id,
+        name: hit._source.ds_name,
+        metadata: hit._source.ds_meta
+      }
+    },
+    
+    ionImage(hit) {
+      return {
+        url: hit._source.ion_image_url
+      };
+    },
+  
+    // fetches data without exposing database IDs to the client
+    peakChartData(hit) {
+      const {sf_adduct, ds_meta} = hit._source;
+      const msInfo = ds_meta.MS_Analysis;
+      const host = config.MOLDB_SERVICE_HOST,
+        instr = lodash.toLower(msInfo.Analyzer),
```

Comment:
no need to use `lodash` here, just use [toLowerCase](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)